docs/about/release-notes.md

```diff
@@ -1,5 +1,35 @@
 # Release Notes
 
+
+## ShapeWorks 6.2.1 - 2022-01-07
+
+### What is new?
+
+#### Studio
+  * Fixed display names in the corner (removing long paths)
+  * Added convert to mesh pipeline for segmentation inputs (can run both image and mesh pipelines)
+  * Fixed #1612 and #1616, bugs regarding recomputing of shape statistics
```

Comment:
Same here. Also, I would remove links to issues and be explicit on what has been fixed.